src/pages/ArtistsCollectionPage.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+import { getUserFollowedArtists } from "../util/spotify";
+import ArtistsCollectionView from "../components/ArtistsCollectionView";
+import PropTypes from "prop-types";
+
+export default function ArtistsCollectionPage(props) {
+  const [followedArtists, setfollowedArtists] = useState(null);
+
+  useEffect(() => {
+    getUserFollowedArtists()
+      .then((data) => {
+        setfollowedArtists(data.artists.items);
+      })
+
+      .catch((err) => {
+        console.log(err);
+      });
+  }, []);
+
+  if (!followedArtists) {
+    return "Loading, bitch...";
+  }
+
+  return (
+    <ArtistsCollectionView
+      followedArtists={followedArtists}
+      artists={props.artists}
+    />
+  );
+}
+
+ArtistsCollectionPage.propTypes = {
+  followedArtists: PropTypes.array,
```

Comment:
Doesn't seem to me that `followedArtists` is a prop you pass into this component. Howwever, it does look like you are using `props.artists`